Adding event and error selector fields (#12921)
ethereum/solidity: Changelog.md

### 0.8.14 (unreleased)

Language Features:

> * General: Extended the language to add the event selector field.

Reviewer: All the items in this section extend the language, so it does not have to be explicitly mentioned.

### 0.8.14 (unreleased)

Language Features:

> * General: Add the event selector field.

Reviewer: ```suggestion
* General: Add `E.selector` for a non-anonymous event `E` to access the 32-byte selector topic.
```

> * General: Errors and Events are visible via other contracts.

Reviewer: ```suggestion
* General: Errors and Events allow qualified access from other contracts.
```
(we should explain what that means in the release post)
